tests/end-to-end/external-solution-integration/internal/scenario/event_mesh/event_mesh.go

```diff
@@ -67,16 +69,15 @@ func (s *Scenario) Steps(config *rest.Config) ([]step.Step, error) {
 				s.applicationGroup, appOperatorClientset.ApplicationconnectorV1alpha1().Applications(),
 				httpSourceClientset.HTTPSources(kymaIntegrationNamespace)),
 		),
-		step.Parallel(
```

Comment:
What is the reasoning behind removing Parallel?

Reply:
It was not reliable under load constraints. This should be also better solved in: https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/issues/kyma-project/kyma/8409